src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -108,4 +112,81 @@ private List<QdmValueSet.Concept> getValueSetConcepts(
     log.info("No Expansion codes are found for the valueSet oid : [{}]", valueSet.getId());
     return List.of();
   }
+
+  // one to call only, one to mutate and build
+  private Bundle retrieveCodeSystemsPage(UmlsUser umlsUser, Integer offset, Integer count) {
```

Comment:
Please re-order methods such that called private methods are either below the calling public method or after all public methods.

It's a stylistic choice that recognizes readers usually end up in a class is by following its exposed methods.